static/docs/user-guide/external-dependencies.md

```diff
@@ -90,7 +90,7 @@ $ dvc run -d remote://example/data.txt \
 Please refer to `dvc remote add` for more details like setting up access
 credentials for certain remotes.
 
-## Using import
+## Using import-url
```

Comment:
we should update this page eventually, and include the regular import also

Reply:
Will do

Reply:
in #487